test/inline/fixtures/layout-template/test.js

```diff
@@ -0,0 +1 @@
+module.exports.fails = "TODO: fix or remove";
```

Comment:
This single test fails because when using `inline templates` there is no component set on the componentContext? @mlrawlings would you mind taking a look at this?